addon-test-support/ember-qunit/test-loader.js

```diff
@@ -57,5 +57,16 @@ export class TestLoader extends AbstractTestLoader {
    @method loadTests
  */
 export function loadTests() {
```

Comment:
Lets see if there are tests for `loadTests()` and see if we should add one for `getTestLoaderInstance()`